src/Mod/Sketcher/Gui/CommandSketcherTools.cpp

```diff
@@ -1815,7 +1815,7 @@ CmdSketcherRectangularArray::CmdSketcherRectangularArray()
     sAppModule      = "Sketcher";
     sGroup          = QT_TR_NOOP("Sketcher");
     sMenuText       = QT_TR_NOOP("Rectangular Array");
-    sToolTipText    = QT_TR_NOOP("Creates a rectangular array pattern of the geometry taking as reference the last selected point");
+    sToolTipText    = QT_TR_NOOP("Creates an rectangular array pattern of the geometry taking as reference the last selected point");
```

Comment:
'not a native english speaker but I could swear it's supposed to be `a`

Reply:
Indeed it should. I never changed this file, Sir.